.gitignore

```diff
@@ -0,0 +1,6 @@
+db.sqlite3
```

Comment:
Thanks for these (long overdue) changes! One thing I'm not sure about is whether it's necessary to include sqlite in the gitignore. Most people who've taken this tutorial aren't experienced with databases, and I'm hoping to use as few unfamiliar terms as possible.